src/marks/image.js

```diff
@@ -61,14 +63,18 @@ export class Image extends Mark {
     this.src = cs;
     this.width = cw;
     this.height = ch;
+    this.sw = width == null && r != null ? 1 : 1 / 2;
+    this.sh = height == null && r != null ? 1 : 1 / 2;
+    this.clipR = r != null && options.clip !== null;
```

Comment:
I think we’ll want to allow **r** to be specified _in addition_ to **width** and **height**, so that, for example, you can have a radius which crops the rectangle (rather than always being a radius of 100%).